pmm-app/tests/pages/qanPage.js

```diff
@@ -457,4 +467,70 @@ module.exports = {
     const pages = await I.grabTextFrom(pagesCount);
     return pages[pages.length - 1];
   },
+
+  addSpecificColumn(columnName) {
+    const column = `//li[contains(text(), '${columnName}')]`;
+    I.waitForVisible(column, 30);
+    I.click(column);
+  },
+
+  verifyAddedColumn(columnName) {
+    const columnHeader = this.getColumn(columnName);
+    I.waitForVisible(columnHeader, 30);
+    I.seeElement(columnHeader);
+  },
+
+  changeMetricInTable(columnName) {
+    const columnHeader = this.getColumn(columnName);
+    I.waitForVisible(columnHeader, 30);
+    I.click(columnHeader);
+  },
+
+  verifyChangedColumn(columnName) {
+    const columnHeader = this.getColumn(columnName);
+    I.waitForInvisible(columnHeader, 30);
+  },
+
+  getColumn(column) {
+    return `//span[contains(text(), '${column}')]`;
+  },
+
+  getRow(row) {
+    return `//td[@class='ant-table-row-cell-break-word']//div[contains(text(), '${row}')]`;
+  },
+
+  async verifyQueryTime(queryTime) {
+    const queryTimeDetial = await I.grabTextFrom(this.fields.queryTimeDetail);
+    assert.equal(queryTimeDetial, queryTime, 'Query times are not same!');
+  },
+
+  //These calculations and verifications needs to be improved https://jira.percona.com/browse/PMM-6140
+  async verifyAvqQueryCount() {
+    const queryTimeDetial = await I.grabTextFrom(this.fields.queryTimeDetail);
+    const queryCountDetail = await I.grabTextFrom(this.fields.queryCountDetail);
+    const result = parseFloat(queryCountDetail) / 300;
+    const roundedResult = Math.round(parseFloat(result) * 10) / 10;
+    //TBD
+  },
+
+  //These calculations and verifications needs to be improved https://jira.percona.com/browse/PMM-6140
+  async verifyAvgQueryTime() {
+    const queryTimeDetail = await I.grabTextFrom(this.fields.queryTimeDetail);
+    const queryCountDetail = await I.grabTextFrom(this.fields.queryCountDetail);
+    const load = await I.grabTextFrom(this.fields.load);
+    const result = (parseFloat(queryCountDetail) * (parseFloat(queryTimeDetail) / 1000)) / 300;
+    const roundedResult = Math.round(parseFloat(result) * 100) / 100;
+    assert.equal(roundedResult, parseFloat(load), 'Load should be same!');
+  },
+
+  //These calculations and verifications needs to be improved https://jira.percona.com/browse/PMM-6140
+  async verifyAvgLockTime() {
+    const avgLoad = await I.grabTextFrom(this.fields.avgLoad);
+    const lockTimeDetail = await I.grabTextFrom(this.fields.lockTimeDetail);
+    const queryCountDetail = await I.grabTextFrom(this.fields.queryCountDetail);
+    const result = (parseFloat(queryCountDetail) * (parseFloat(lockTimeDetail) / 1000000)) / 300;
```

Comment:
parsFloat's signature is not respected -> should be `parseFloat(lockTimeDetail, 10)`